docs/README.md

```diff
@@ -22,65 +22,103 @@ cd cluster-api-provider-nested
+capn-nested-control-plane-system   capn-nested-control-plane-controller-manager-698c444c6d-nhddn   2/2     Running   0          4m45s
+capn-system                        capn-controller-manager-7f9757b67f-cp8d9                        2/2     Running   0          4m43s
+cert-manager                       cert-manager-5597cff495-gwgr5                                   1/1     Running   0          5m7s
+cert-manager                       cert-manager-cainjector-bd5f9c764-vvbjf                         1/1     Running   0          5m7s
+cert-manager                       cert-manager-webhook-5f57f59fbc-ccg5k                           1/1     Running   0          5m7s
+kube-system                        coredns-74ff55c5b-9pspc                                         1/1     Running   0          6m16s
+kube-system                        coredns-74ff55c5b-nqnk9                                         1/1     Running   0          6m16s
+kube-system                        etcd-capn-control-plane                                         1/1     Running   0          6m29s
+kube-system                        kindnet-9g9z4                                                   1/1     Running   0          6m16s
+kube-system                        kube-apiserver-capn-control-plane                               1/1     Running   0          6m29s
+kube-system                        kube-controller-manager-capn-control-plane                      1/1     Running   0          6m29s
+kube-system                        kube-proxy-jl46r                                                1/1     Running   0          6m16s
+kube-system                        kube-scheduler-capn-control-plane                               1/1     Running   0          6m29s
+local-path-storage                 local-path-provisioner-78776bfc44-qcx49                         1/1     Running   0          6m16s
+```
 
-Next, we will deploy the CAPN related CRDs and controllers.
+### Set clustername (in our example, we set clustername to `cluster-sample`)
 
 ```shell
-kubectl apply -f out/cluster-api-provider-nested-components.yaml 
+export CLUSTER_NAME=cluster-sample
 ```
 
-### Apply Sample Cluster
+### Generate custom resource (`Cluster`, `NestedCluster` etc) and apply to our cluster
 
 ```shell
-kubectl apply -f config/samples/
+# ./bin/clusterctl generate cluster ${CLUSTER_NAME} --infrastructure=nested:v0.1.0
 ```
 
 ### Get `KUBECONFIG`
 
-We will use the `clusterctl` command-line tool to generate the `KUBECONFIG`, which 
-will be used to access the nested controlplane later.
-
 ```shell
-cd cluster-api
-make clusterctl
-./bin/clusterctl get kubeconfig cluster-sample > ../kubeconfig
-cd ..
+# ./bin/clusterctl get kubeconfig ${CLUSTER_NAME} > kubeconfig
+```
+
+Note you might see following error, wait for a while and retry
+```
+Error: "cluster-sample-kubeconfig" not found in namespace "default": secrets "cluster-sample-kubeconfig" not found
 ```
```

Comment:
Maybe above this section, we can add a note saying to run `kubectl get cluster -w` and wait for the `Status` to say `Ready`.

Reply:
Can we just say: 

```This error means that the status of the cluster is not ready. Please wait until the cluster is ready and try again.```

Reply:
>>>kubectl get cluster -w

the status stuck at 
```
NAME             PHASE
cluster-sample   Provisioned
```
for long time..

Reply:
it should be running per cluster-api project
I guess some pre-requirement not complete so status of cluster is not running ...need find reason later on

Reply:
Yeah, there was a bug with the published version actually :/ so that object doesn't get updated properly, I fixed it in #136 maybe we can use `kubectl get nestedcluster -w` instead since that updates properly and then when that other PR is merged we can switch this back to the cluster resource.